sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-disbursements-report-bc-grants.sql

```diff
@@ -0,0 +1,75 @@
+UPDATE
+    sims.report_configs
+SET
+    report_sql = (
+    '(
+      select
+        to_char(dr.disburse_date, ''YYYY-MM-DD'') as "Date of Disbursement",
+        dr.student_sin as "SIN",
+        app.application_number as "Application Number",
+        ds.document_number as "Certificate Number",
+        drv.grant_type as "Funding Code",
+        drv.grant_amount as "Disbursement Amount"
+      from
+        sims.disbursement_receipts dr
+        inner join sims.disbursement_receipt_values drv on drv.disbursement_receipt_id = dr.id
+        inner join sims.disbursement_schedules ds on ds.id = dr.disbursement_schedule_id
+        inner join sims.student_assessments sa on sa.id = ds.student_assessment_id
+        inner join sims.applications app on app.id = sa.application_id
+        inner join sims.education_programs_offerings epo on epo.id = sa.offering_id
+      where
+        epo.offering_intensity = any(:offeringIntensity)
+        and dr.disburse_date between :startDate
+        and :endDate
+      union
+      all
+      select
+        to_char(dr.disburse_date, ''YYYY-MM-DD'') as "Date of Disbursement",
+        dr.student_sin as "SIN",
+        app.application_number as "Application Number",
+        ds.document_number as "Certificate Number",
+        dv.value_code as "Funding Code",
+        dv.effective_amount as "Disbursement Amount"
+      from
+        sims.disbursement_receipts dr
+        inner join sims.disbursement_receipt_values drv on drv.disbursement_receipt_id = dr.id
+        inner join sims.disbursement_schedules ds on ds.id = dr.disbursement_schedule_id
+        inner join sims.disbursement_values dv on dv.disbursement_schedule_id = ds.id
+        inner join sims.student_assessments sa on sa.id = ds.student_assessment_id
+        inner join sims.applications app on app.id = sa.application_id
+        inner join sims.education_programs_offerings epo on epo.id = sa.offering_id
+      where
+        epo.offering_intensity = any(:offeringIntensity)
```

Comment:
I missed to mention in this. The BCSG will be available only for receipts of funding_type which is NOT `FE`. That could be the first condition after offering intensity.

```sql
and dr.funding_type <> 'FE'
```